tests/test_L031_insecure_file_permission_graph.py

```
@@ -0,0 +1,430 @@
+        assert rule.rule_id == "L031"
+        assert rule.enabled is True
+
+    def test_match_file_module_with_mode(self, rule: InsecureFilePermissionGraphRule) -> None:
+        """Matches file module task with mode parameter.
+
+        Args:
+            rule: Rule instance under test.
+        """
+        g, nid = _make_file_task(
+            module="ansible.builtin.file",
+            module_options={"path": "/tmp/test", "mode": "0644"},
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)